### PR TITLE
SDDSIP-2138: CONTACT DETAILS -  Copy changes batch 1

### PR DESCRIPTION
**Pull request by richard-mohammed-def**
https://eaflood.atlassian.net/browse/SDDSIP-2138

### CODE REVIEW

**Review by jonathangoulding — APPROVED** [review]

File: packages/web-service/src/pages/contact/additional-contacts/additional-ecologist/add-additional-ecologist.njk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends "yes-no.njk" %}
-{% set title = 'Do you want to add an alternative ecologist contact?' %}
+{% set title = 'Do you want to add an alternate ecologist?' %}
 {% set radioHint = 'This alternative contact must be someone who is authorised to deal with this application on behalf of the named ecologist.' %}
```
Comment (StuAA78) [inline packages/web-service/src/pages/contact/additional-contacts/additional-ecologist/add-additional-ecologist.njk:3]: AC4 in the ticket states that this hint text needs to be changed to `This person will not hold the licence, but will receive messages about this application.`

File: packages/web-service/src/pages/contact/additional-contacts/additional-applicant/add-additional-applicant.njk [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {% extends "yes-no.njk" %}
-{% set title = 'Do you want to add an alternative contact?' %}
+{% set title = 'Do you want to add an alternate applicant?' %}
 {% set radioHint = 'This alternative contact must be someone who is authorised to deal with this application on behalf of the intended licence-holder.' %}
```
Comment (StuAA78) [inline packages/web-service/src/pages/contact/additional-contacts/additional-applicant/add-additional-applicant.njk:3]: AC3 of the ticket _should_ state that this hint text should be changed to `This person will not hold the licence, but will receive messages about this application.` as per the prototype and prototype screenshot. It doesn't, but can we make the change here anyway. I'll update the ticket accordingly and throw a comment on it to explain.

**Review by StuAA78 — CHANGES_REQUESTED** [review]
A couple of small changes needed to hint text.

File: packages/web-service/src/pages/contact/additional-contacts/additional-ecologist/add-additional-ecologist.njk (hunk 1 above)
Comment (richard-mohammed-def) [reply packages/web-service/src/pages/contact/additional-contacts/additional-ecologist/add-additional-ecologist.njk:3]: Will do. Well spotted. I honestly didn't see that at all

File: packages/web-service/src/pages/contact/additional-contacts/additional-applicant/add-additional-applicant.njk (hunk 2 above)
Comment (richard-mohammed-def) [reply packages/web-service/src/pages/contact/additional-contacts/additional-applicant/add-additional-applicant.njk:3]: Will make the change

**Review by StuAA78 — APPROVED** [review]
Nice one 👍

**Pull request merged**